frontend/src/component/Account/UserInfo.jsx

```diff
@@ -2,44 +2,104 @@ import React from 'react'
 import { translate } from 'react-i18next'
 import { getUserProfile, FETCH_CONFIG } from '../../util/helper.js'
 import { Avatar, AVATAR_SIZE } from 'tracim_frontend_lib'
+import { Popover, PopoverBody } from 'reactstrap'
+import { isMobile } from 'react-device-detect'
 
 require('./UserInfo.styl')
 
-export const UserInfo = props =>
-  <div className='userinfo mr-5 ml-5 mb-5' data-cy='userinfo'>
-    <div className='userinfo__avatar' data-cy='userinfo__avatar'>
-      <Avatar
-        size={AVATAR_SIZE.BIG}
-        user={props.user}
-        apiUrl={FETCH_CONFIG.apiUrl}
-      />
-    </div>
-
-    <div className='userinfo__wrapper'>
-      <div className='userinfo__name primaryColorFont' data-cy='userinfo__name'>
-        {`${props.user.publicName}`}
-      </div>
+export class UserInfo extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      popoverFullNameOpen: false,
+      popoverUsernameOpen: false
+    }
+  }
+
+  handleTogglePopoverFullName = () => {
+    this.setState(prevState => ({
+      popoverFullNameOpen: !prevState.popoverFullNameOpen
+    }))
+  }
+
+  handleTogglePopoverUsername = () => {
+    this.setState(prevState => ({
+      popoverUsernameOpen: !prevState.popoverUsernameOpen
+    }))
+  }
+
+  render () {
+    const { props } = this
+
+    return (
+      <div className='userinfo mr-5 ml-5 mb-5' data-cy='userinfo'>
+        <div className='userinfo__avatar' data-cy='userinfo__avatar'>
+          <Avatar
+            size={AVATAR_SIZE.BIG}
+            user={props.user}
+            apiUrl={FETCH_CONFIG.apiUrl}
+          />
+        </div>
+
+        <div className='userinfo__wrapper'>
+          <div div className='userinfo__name primaryColorFont' data-cy='userinfo__name'>
+            <span href='#' id='popoverFullName'>{`${props.user.publicName}`}
+            </span>
```

Comment:
What is the reason for the `href='#'` in the span?

Suggestion:
```suggestion
            <span href='#' id='popoverFullName'>
              {`${props.user.publicName}`}
            </span>
```
Note: my suggestion is applicable to the other spans, not required, just a suggestion because I think it improves code readability

Reply:
You're right it's not useful :) !